### DISCUSSION TITLE
Vercel deployment error

### DISCUSSION THREAD

**Discussion by akanz** [vercel/community#471] [Help] [answered]
Hey! I'm having the error below when deploying my Nextjs app to vercel. It states the error is from the platform. any help? 
I opened the issue under next but It should be opened here too.

<img width="1440" alt="Screenshot 2022-03-10 at 12 47 09" src="https://user-images.githubusercontent.com/33222698/157867528-9e5a7c7d-6274-402f-aa81-79031d1813b4.png">

**Comment by RizBizKits** [comment]
👋 Hey! Thank you for your patience. Can you let me know if you're still encountering this issue at this time? Any deployment URLs that you can share? What was the git diff between your latest failing and non-failing deployments? Happy to take a closer look 🕵️‍♀️

**Comment by akanz** [comment, marked as answer]
Hey, thanks for reaching out.

Apparently My environmental variables were that of a CRA and not of Nextjs.
That fixed the issue.
